packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -243,6 +251,14 @@ export interface VideoGalleryProps {
    * @defaultValue 'HorizontalBottom'
    */
   overflowGalleryPosition?: OverflowGalleryPosition;
+  /* @conditional-compile-remove(click-to-call) */
+  /**
+   * Determines the aspect ratio of local video tile in the video gallery.
+   * @remarks 'matchStream' will be responsive to the screen orientation and will change between 9:16 (portrait) and
```

Comment:
Yes from my understanding through the only time the stream aspect ratio will change from a portrait view on mobile to landscape view is when the device orientation changes. the stream updates from 9:16 to 16:9 when you rotate the device. should I update the comment so it is referring to the stream here even though we are truthfully in the implementation using the device aspect ratio? 

there currently is not a good way for us to get the aspect ratio in a performant way from the SDK. media stats API is not a good solution here in my opinion since we have to set up something to ping the stats every so often so the device could rotate and the tile wont update for a second or two depending on the interval set.

Reply:
Yeah, matchStream is a bit of lie then, matchDevice might be more accurate.. but the aspect ratio doesn't match the device, we just estimate it.
I would rename to matchDevice if we can exactly match the device, perhaps `followDeviceOrientation` ?

Reply:
I like the `followDeviceOrientation` I will make that update I think that makes sense